### PR TITLE
chore: Update contributors across the project

### PR DESCRIPTION
**Pull request by rogeruiz**
# Summary

This patch series includes some changes that are needed to address @Igarfinkle's comment concerning updating the contributors in places outside of the scope of the `CODEOWNERS` and current **Active Maintainer** section in our documentation. 

I've decided to open this separately and remove the commits from #1968 as they're outside of the scope of the initial PR. This is currently in **Draft** as there may be other changes that need to be made to other files besides adding @Igarfinkle information.

## Related Issues or PRs

- #1968 
- https://github.com/trussworks/react-uswds/pull/1968#issuecomment-1092006258

## How To Test

I'm not sure the best way to test this as I'm new to maintaining this project. If there's something I missed, please edit this comment or add an additional comment and I can edit this description.

>Danger: Ignore "Do not make changes to package.json around contributors. - This project only uses .all-contributorsrc for tracking contributors."

### CODE REVIEW

File: package.json
Comment (rogeruiz) [inline package.json:52]: I understand that the four people above are no longer working at Truss, but taking a look at the history of the repository, these folks did contribute to the project. The property here is `contributors` and not `active_contributors` so I'm not sure if removing them makes sense. But I'm okay to treat this as `active_contributors` if that's what the goal of this change is.
cc: @Igarfinkle

**Comment by rogeruiz** [comment]
Also, @Igarfinkle given the original intention of your changes here, should we be removing folks who are no longer at Truss from the `.all-contributorsrc` file as well?

```diff
diff --git a/.all-contributorsrc b/.all-contributorsrc
index fdfdf21..72c8758 100644
--- a/.all-contributorsrc
+++ b/.all-contributorsrc
@@ -111,15 +111,6 @@
         "code"
       ]
     },
-    {
-      "login": "christopherhuii",
-      "name": "Christopher Hui",
-      "avatar_url": "https://avatars.githubusercontent.com/u/8367504?v=4",
-      "profile": "https://github.com/christopherhuii",
-      "contributions": [
-        "code"
-      ]
-    },
     {
       "login": "jim",
       "name": "Jim Benton",
```

**Comment by haworku** [comment]
Agree - all contributors get to stay! In fact, I would vote we remove the `contributors` key entirely from the `package.json` and instead lean into the [allcontributers](https://github.com/all-contributors/all-contributors) file (which was added later on). It feels like there is unnecessary a duplication here.

Folks who contribute once are still contributors. Adjusting the CODEOWNERS when folks move on from Truss is the only thing we need to change.

**Comment by brandonlenz** [comment]
I'm on the same page as @haworku here. I'm not aware of anything specific `contributors` in `package.json` gets us that `allcontributors` doesn't already convey. IMO, less duplication = less overhead.

File: package.json
Comment (Igarfinkle) [reply package.json:52]: Oh, of course! My mistake, we should absolutely include them.

Comment (Igarfinkle) [reply package.json:52]: fixed

**Comment by rogeruiz** [comment]
I ran a force-push here to rebase my changes rather than merge them. But if that's not the right way to do things let me know. thanks!

File: .all-contributorsrc
Comment (haworku) [inline .all-contributorsrc:198]: I think I'm already in there higher up? This might be a dupe.

**Review by haworku — DISMISSED** [review]

**Comment by rogeruiz** [comment]
@haworku sorry I think I dismissed the review you had made when I added the label 😕

File: .all-contributorsrc
Comment (rogeruiz) [reply .all-contributorsrc:188]: oh my. I really needed my coffee.

**Review by haworku — APPROVED** [review]

**Review by brandonlenz — DISMISSED** [review]

**Comment by Igarfinkle** [comment]
Danger: Ignore "Do not make changes to package.json around contributors."

**Comment by rogeruiz** [comment]
yeah @Igarfinkle I'm thinking of rebasing and removing those commits that adds things to `package.json` and see if that clears things up for Danger.

**Comment by Igarfinkle** [comment]
> Danger: Ignore "Developer Specific file shouldn't be changed"

**Comment by Igarfinkle** [comment]
> Danger: Ignore "Do not make changes to package.json around contributors."

**Comment by Igarfinkle** [comment]
@rogeruiz apparently there's an "allow a Fail just this once", which I'm trying to trigger

**Comment by rogeruiz** [comment]
@Igarfinkle i wonder if this is supposed to be in my original PR body. You're free to edit my description for the PR.

**Comment by brandonlenz** [comment]
> @Igarfinkle i wonder if this is supposed to be in my original PR body. You're free to edit my description for the PR.

I tried adding it there first. Didn't seem to help, BUT it looks like danger just keeps referencing it's own first comment, so I edited that comment to remove the fail to see if danger will care/re-add it.

**Comment by brandonlenz** [comment]
Nope, Danger, updated it's own comment, so the ignore is not working 😢

**Comment by brandonlenz** [comment]
Trying again removing the <i> from the ignore message in the PR body in case that's messing with Danger's lookup

**Review by Igarfinkle — APPROVED** [review]

**Pull request merged**
